UserManagement/views.py

```diff
@@ -20,10 +20,10 @@
 def deleteUser(request):
     body_unicode = request.body.decode('utf-8')
     try:
-        body_data = json.loads(body_unicode)
-        user_id = body_data["user_id"]
+        user_id = int(request.GET.get("user_id"))
         user = User.objects.get(id = user_id)
-        user.delete()
+        user.is_deleted = True
+        user.save()

```

Comment:
For the deleteUser Rest API, what's the point of using a boolean to represent when a user is deleted to compared to just removing it from the database? For instance, this makes sense for the item db because items can come and go when their inventory reaches 0, but if a user needs to be removed should it not just be deleted?